app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,22 @@
+class VotesController < ApplicationController
+
+  def create
+    @vote = Vote.new(vote_params)
+    @vote.user = current_user
+    if params[:up_vote]
+      @vote.vote_type = 1
+    elsif params[:down_vote]
+      @vote.vote_type = -1
```

Comment:
Seems like this could be handled a little more cleanly with vote params that include `vote_type`. Doesn't need to block you for this PR, but we should look at it together.

Reply:
Good idea